core/urls.py

```diff
@@ -16,8 +16,19 @@
 """
 
 from django.contrib import admin
+from django.urls import include
 from django.urls import path
+from drf_spectacular.views import SpectacularAPIView, SpectacularRedocView, SpectacularSwaggerView
 
 urlpatterns = [
     path("admin/", admin.site.urls),
+    path("__debug__/", include("debug_toolbar.urls")),
+    path("/api/auth/", include("djoser.urls")),
+    path("/api/auth/", include("djoser.urls.jwt")),
```

Comment:
There shouldn't be leading slashes in `api/auth/` endpoints.  (_Fixed in the next pull request._)
```suggestion
    path("api/auth/", include("djoser.urls")),
    path("api/auth/", include("djoser.urls.jwt")),
```